File: RLE.API/Models/TrainingClass.cs
using System;
using System.Collections.Generic;

namespace RLE.API.Models
{
    public partial class TrainingClass
    {

        public int Id { get; set; }
        public string Name { get; set; }
        public int? RegionId { get; set; }
        public int? DepartmentId { get; set; }
        public int? CityId { get; set; }
        public int? MunicipalityId { get; set; }
        public int TrainingId { get; set; }
        public DateTime? EtastartDate { get; set; }
        public DateTime? EtaendDate { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public byte Status { get; set; }
        public byte Active { get; set; }

        public  City City { get; set; }
        public  Department Department { get; set; }
        public  Municipality Municipality { get; set; }
        public  Region Region { get; set; }
        public  Training Training { get; set; }
        // public  ICollection<TrainerClass> TrainerClass { get; set; }
    }
}

File: RLE-SPA/src/app/_models/trainingClass.ts
import { City } from './city';
import { User } from './user';

export class TrainingClass {
  id: number;
  name: string;
  regionId: number;
  trainingId: number;
  sessionName: string;

  regionName: string;
  departmentId: number;
  departmentName: string;
  startDate: string;
  endDate: string;
  cityId: number;
  status: number;
  Active: boolean;
  cityName; string;
  totalTrainers: number;
  totalParticipants: number;
  trainers: User[];
  cities: City[];
  participants: User[];
  trainerIds: [];
  trainingDate: Date;
  totalTrained: number;
  totalPhotos: number;
  summarized: boolean;
}
